README.md

```diff
@@ -193,6 +193,76 @@ public class SomeOtherTest {
     }
 ```
 
+##### Mapping ports
+
+Often there is a need to connect to a container via a certain port. To achieve this goal, you need to know the mapped 
+port of the port that was specified from the Java code. To get the mapped port use the `getMappedPort(...)` method
+of testcontainers API. 
+
+As an example, consider the following Java code:
+
+```java
```

Comment:
It's better to have it in test code and put the link here

Reply:
Fixed